system/mrm_handler/src/mrm_handler/mrm_handler_core.cpp

```diff
@@ -397,11 +397,12 @@ void MrmHandler::updateMrmState()
 
   // Get mode
   const bool is_auto_mode = isAutonomous();
```

Comment:
@kyoichi-sugahara 
Could you rename the followings in terms of readability?
- `is_auto_mode` -> `is_control_mode_autonomous`
- `is_operation_mode_auto_mode` -> `is_operation_mode_autonomous`
- `isAutonomous()` -> `isControlModeAutonomous()

Reply:
@veqcc 
Addressed in https://github.com/autowarefoundation/autoware.universe/pull/7784/commits/18da2cb123f9e0c140803042a1e772088ccd5b9e
Can you approve the PR?

Reply:
Thank you!

Reply:
@kyoichi-sugahara 
You may have to change the function name here:
```
bool MrmHandler::isAutonomous()
{
  using autoware_vehicle_msgs::msg::ControlModeReport;
  auto mode = sub_control_mode_.takeData();
  if (mode == nullptr) return false;
  return mode->mode == ControlModeReport::AUTONOMOUS;
}
```

Reply:
@veqcc 
Sorry forgot to change and addressed in https://github.com/autowarefoundation/autoware.universe/pull/7784/commits/7297667b9f654875a5b5b1178f989ae60f670529